app/components/app_activity_log_component.rb

```diff
@@ -110,12 +110,21 @@ def triage_events
   end
 
   def vaccination_events
-    vaccination_records.recorded.map do
+    vaccination_records.recorded.map do |vaccination_record|
+      vaccine = vaccination_record.vaccine
+
+      title =
+        if vaccination_record.administered?
+          "Vaccinated with #{helpers.vaccine_heading(vaccine)}"
+        else
+          "Not vaccinated with #{helpers.vaccine_heading(vaccine)}"
```

Comment:
Not sure about listing the vaccine here? It may require rework for flu, where you have two possible products (nasal and injection). Maybe we should mention the programme instead? I'll check with the design team.

Reply:
Yeah, I did wonder about that myself, I think not including the vaccine would be reasonable.